qt_ui/windows/mission/flight/settings/QFlightStartType.py

```diff
@@ -43,6 +50,29 @@ def __init__(self, package_model: PackageModel, flight: Flight):
         )
         self.setLayout(self.layout)
 
+        pilots_changed.connect(self.on_pilot_selected)
+
+    def on_pilot_selected(self):
+        # Pilot selection detected. If this is a player flight, set start_type
+        # as configured for players in the settings.
+        # Otherwise, set the start_type as configured for AI.
+        # https://github.com/dcs-liberation/dcs_liberation/issues/1567
+
+        if self.flight.roster.player_count > 0:
+            self.flight.start_type = (
+                self.flight.coalition.game.settings.default_start_type_client
+            )
+        else:
+            self.flight.start_type = (
+                self.flight.coalition.game.settings.default_start_type
+            )
+
+        for i, st in enumerate([b for b in ["Cold", "Warm", "Runway", "In Flight"]]):
+            if self.flight.start_type.value == st:
+                self.start_type.setCurrentIndex(i)
```

Comment:
This can probably be replaced as well by `self.start_type.setCurrentText(self.flight.start_type.value)`

Reply:
Done 👍